access/types/stdasync.py

```diff
@@ -390,58 +361,44 @@ def _acceptSubmission(request, course, exercise, post_url, sdir):
     _acceptSubmission.counter += 1
 
     # Order container for grading.
-    if container_flag:
-        c = _requireContainer(exercise)
-
-        course_extra = {
-            "key": course["key"],
-            "name": course["name"],
-        }
-        exercise_extra = {
-            "key": exercise["key"],
-            "title": exercise.get("title", None),
-            "resources": c.get("resources", {}), # Unofficial param, implemented differently later
-            "require_constant_environment": c.get("require_constant_environment", False) # Unofficial param, implemented differently later
-        }
-        if exercise.get("personalized", False):
-            exercise_extra["personalized_exercise"] \
-                = select_generated_exercise_instance(course, exercise, uids, attempt)
-            if settings.ENABLE_PERSONAL_DIRECTORIES:
-                exercise_extra["personal_directory"] \
-                    = user_personal_directory_path(course, exercise, uids)
-
-        sid = os.path.basename(sdir)
-        write_submission_meta(sid, {
-            "url": surl,
-            "dir": sdir,
-            "course_key": course["key"],
-            "exercise_key": exercise["key"],
-            "lang": translation.get_language(),
-        })
-        r = invoke([
-            settings.CONTAINER_SCRIPT,
-            sid,
-            request.scheme + "://" + request.get_host(),
-            c["image"],
-            os.path.join(DIR, course["key"], c["mount"]),
-            sdir,
-            c["cmd"],
-            json.dumps(course_extra),
-            json.dumps(exercise_extra),
-        ])
-        LOGGER.debug("Container order exit=%d out=%s err=%s",
-            r["code"], r["out"], r["err"])
-        qlen = 1
-
-    # Queue in celery & rabbitmq for chroot sandbox actions.
-    else:
-        tasks.grade.delay(course["key"], exercise["key"],
-            translation.get_language(), surl, sdir, uids, attempt)
-        qlen = queue_length()
-        LOGGER.debug("Submission of %s/%s, queue counter %d, queue length %d",
-            course["key"], exercise["key"], _acceptSubmission.counter, qlen)
-        if qlen >= settings.QUEUE_ALERT_LENGTH:
-            LOGGER.error("Queue alert, length: %d", qlen)
+    c = _requireContainer(exercise)
+
+    course_extra = {
+        "key": course["key"],
+        "name": course["name"],
+    }
+    exercise_extra = {
+        "key": exercise["key"],
+        "title": exercise.get("title", None),
+        "resources": c.get("resources", {}), # Unofficial param, implemented differently later
+        "require_constant_environment": c.get("require_constant_environment", False) # Unofficial param, implemented differently later
+    }
+    if exercise.get("personalized", False):
+        exercise_extra["personalized_exercise"] \
+            = select_generated_exercise_instance(course, exercise, uids, attempt)
```

Comment:
You could add to the git commit message that the personal directory feature has been removed (in addition to the chroot sandbox).

Reply:
I think I will handle the removal of the personal directory in a different PR. It seems like there are a couple of files that need to be modified/removed.